tests/func/experiments/test_experiments.py

```diff
@@ -606,17 +606,12 @@ def test_run_env(tmp_dir, dvc, scm, mocker):
     dump_run_env = dedent(
         """\
         import os
-        from dvc_studio_client.env import STUDIO_REPO_URL
```

Comment:
With this PR, we save the studio config as an additional kwarg when stashing the experiment instead of saving environment variables. Need to add a test for this behavior.

Reply:
Since we reverted to using env vars, updating the existing tests should be good enough I think